src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_BALL_COUNT = 100;
+
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor();
+        int number = new Random().nextInt(MAX_BALL_COUNT + 1);
```

Comment:
same, let's make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method.